### ISSUE TITLE
[3.7.1] Won't set Chromium 'release_version' on Linux

### ISSUE THREAD

**Issue by camador** [titusfortner/webdrivers#51]
Hi,

I'm getting this error when running specs with Capybara:

```ruby
NoMethodError:
            undefined method `<' for nil:NilClass

# /home/user/.rvm/gems/ruby-2.5.3@gemset/gems/webdrivers-3.7.1/lib/webdrivers/chromedriver.rb:22:in `latest_version'
# /home/user/.rvm/gems/ruby-2.5.3@gemset/gems/webdrivers-3.7.1/lib/webdrivers/common.rb:37:in `desired_version'
# /home/user/.rvm/gems/ruby-2.5.3@gemset/gems/webdrivers-3.7.1/lib/webdrivers/common.rb:16:in `update'
# /home/user/.rvm/gems/ruby-2.5.3@gemset/gems/webdrivers-3.7.1/lib/webdrivers/selenium.rb:7:in `driver_path'
# /home/userr/.rvm/gems/ruby-2.5.3@gemset/gems/selenium-webdriver-3.141.0/lib/selenium/webdriver/chrome/driver.rb:39:in `initialize'
# /home/userr/.rvm/gems/ruby-2.5.3@gemset/gems/selenium-webdriver-3.141.0/lib/selenium/webdriver/common/driver.rb:44:in `new'
# /home/userr/.rvm/gems/ruby-2.5.3@gemset/gems/selenium-webdriver-3.141.0/lib/selenium/webdriver/common/driver.rb:44:in `for'
# /home/user/.rvm/gems/ruby-2.5.3@gemset/gems/selenium-webdriver-3.141.0/lib/selenium/webdriver.rb:86:in `for'
# /home/user/.rvm/gems/ruby-2.5.3@gemset/gems/capybara-2.18.0/lib/capybara/selenium/driver.rb:23:in `browser'
.
.
.
```

Debugging the file you can see the regexp in method `chrome_version` is not working propperly when the Chromium version has a dot before the actual version number:
```ruby
[18, 27] in /home/user/.rvm/gems/ruby-2.5.3@gemset/gems/webdrivers-3.7.1/lib/webdrivers/chromedriver.rb
   18:       def latest_version
   19:         raise StandardError, 'Can not reach site' unless site_available?
   20: 
   21:         # Versions before 70 do not have a LATEST_RELEASE file
   22:         byebug
=> 23:         return Gem::Version.new('2.46') if release_version < '70.0.3538'
   24: 
   25:         # Latest webdriver release for installed Chrome version
   26:         release_file     = "LATEST_RELEASE_#{release_version}"
   27:         latest_available = get(URI.join(base_url, release_file))
(byebug) release_version
nil
(byebug) chrome_version[/\d+\.\d+\.\d+/]
nil
(byebug) chrome_on_linux
"Using PPAPI flash.\n73.0.3683.75"
(byebug) "Using PPAPI flash.\n73.0.3683.75"[/(\d|\.)+/]
"."
(byebug) "Using PPAPI flash\n73.0.3683.75"[/(\d|\.)+/]
"73.0.3683.75"

# Testing modified regexp
(byebug) "Using PPAPI flash.\n73.0.3683.75"[/\d(\d|\.)+/]
"73.0.3683.75"
(byebug) "Using PPAPI flash\n73.0.3683.75"[/\d(\d|\.)+/]
"73.0.3683.75"
(byebug) "Using PPAPI flash 73.0.3683.75"[/\d(\d|\.)+/]
"73.0.3683.75"
```
I'm runnig Linux Mint 19.1 with Chromium Version 73.0.3683.75 (Official Build) Built on Ubuntu.

Thanks

**Comment by kapoorlakshya** [comment]
Hi @camador, the fix for this was merged yesterday and will be available in v3.7.2 soon. Please see #43  for details.

**Comment by camador** [comment]
Ooops, sorry for posting solved issues. Thanks
